Sources/ForageSDK/ForageSDK.swift

```diff
@@ -41,8 +41,17 @@ public class ForageSDK {
         LDManager.shared.initialize(self.environment)
         // TODO: Maybe move this shared logger call!
         VGSCollectLogger.shared.disableAllLoggers()
+       
         self.service = LiveForageService()
+        SentrySDK.start { options in
+            options.dsn = "https://8fcdd8dc94aa892ed8fd4cdb20db90ee@o921422.ingest.sentry.io/4505665631813632"
+            options.debug = true 
```

Comment:
this has to be false

Reply:
075f06997e0cf89acd5a7e53654297e54651b3ee